.github/workflows/build.yaml

```diff
@@ -0,0 +1,136 @@
+              env:
+                GITHUB_ACTOR: ${{github.actor}}
+                GITHUB_TOKEN: ${{ github.token }}
+                ORG_GRADLE_PROJECT_signingKeyId:  ${{ secrets.GPG_KEY }}
+                ORG_GRADLE_PROJECT_signingKey: ${{ secrets.GPG_SECRET }}
+                ORG_GRADLE_PROJECT_signingPassword: ${{ secrets.GPG_PASSPHRASE }}
+              run: |
+               gradle -b galasa-parent/build.gradle check publish \
+               -Dorg.gradle.jvmargs=-Xmx5120M \
+               -PsourceMaven=https://repo.maven.apache.org/maven2/  \
+               -PcentralMaven=https://repo.maven.apache.org/maven2/ \
+               -PtargetMaven=${{ github.workspace }}/repo  -PjacocoEnabled=true -PisMainOrRelease=true
+              
+            - name: Login to Github Container Registry
+              uses: docker/login-action@v3
+              with:
+                registry: ${{ env.REGISTRY }}
+                username: ${{ github.actor }}
+                password: ${{ secrets.GITHUB_TOKEN }}
+       
+            - name: Extract metadata for framework image
+              id: metadata
+              uses: docker/metadata-action@9ec57ed1fcdbf14dcef7dfbe97b2010124a938b7
+              with:
+                images: ${{ env.REGISTRY }}/${{ env.NAMESPACE }}/framework-maven-artefacts
+       
+            - name: Build Framework image for development Maven registry
+              id: build
+              uses: docker/build-push-action@v5
+              with:
+                context: .
+                file: dockerfiles/dockerfile.framework
+                push: true
+                tags: ${{ steps.metadata.outputs.tags }}
+                labels: ${{ steps.metadata.outputs.labels }}
+                build-args: |
+                  dockerRepository=harbor.galasa.dev
+                  tag=${{ env.IMAGE_TAG }}
+            
+            - name: Recycle application in ArgoCD
+              env: 
+                  ARGOCD_AUTH_TOKEN: ${{ secrets.ARGOCD_TOKEN }}
+              run: |
+                  docker run --env ARGOCD_AUTH_TOKEN=${{ env.ARGOCD_AUTH_TOKEN }} --rm -v ${{ github.workspace }}:/var/workspace ghcr.io/galasa-dev/argocdcli:main app actions run gh-maven-repos restart --kind Deployment --resource-name framework-gh --server argocd.galasa.dev
+                    
+            
+    Build-Documentation:
+        name: Build framework using openapi2beans and gradle
+        runs-on: ubuntu-latest
+
+        steps:
+            - name: Checkout Code
+              uses: actions/checkout@v4
+            
+            - name: Setup JDK 
+              uses: actions/setup-java@v4
+              with:
+                java-version: '11'
+                distribution: 'semeru'
+            
+            - name: Install swagger cli
+              run: |
+               wget https://repo1.maven.org/maven2/io/swagger/swagger-codegen-cli/3.0.0-rc0/swagger-codegen-cli-3.0.0-rc0.jar -O swagger-codegen-cli.jar

```

Comment:
Can we change this to `https://repo1.maven.org/maven2/io/swagger/codegen/v3/swagger-codegen-cli/3.0.41/swagger-codegen-cli-3.0.41.jar` so it's the same as our current builds? Just for consistency.

Reply:
done